<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScalesDlg</class>
 <widget class="QWidget" name="ScalesDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>颜色</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>23</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>线条色：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>61</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文本色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_linecolorbtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_textcolorbtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>坐标轴</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>主刻度分割数：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_mainspin">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>副刻度分割数：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_subspin">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>251</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>文本</string>
   </property>
   <widget class="QCheckBox" name="m_showtextcheck">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>显示刻度值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>91</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>字体大小：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>小数位：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pointnumcmb">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="m_minedit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_maxedit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="QComboBox" name="m_fontsizecmb">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="m_textdirectioncmb">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文本方向：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>刻度方向</string>
   </property>
   <widget class="QRadioButton" name="m_radio1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>左</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_radio2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>右</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_radio3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>上</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_radio4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>下</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
